### DISCUSSION TITLE
Строительный сектор в гос. структуре

### DISCUSSION THREAD

**Discussion by Talier** [totumonline/totum-ru-issues-and-discussions#113] [Сделанные проекты]
Основным набором инструментов является контроль ответственных за объекты. Выезд на объект и ведения отчета.
Второстепенными являются:
- Муниципальные контракты и связка с объектами или с частью исполнения контрактов
- Статьи финансирования муниципальных контрактов
- Полностью вся информация по объекту
- Архивы объектов 
https://skr.sh/sDXTxqSPVQo?a

**Comment by mihail-cornev** [comment]
> * Архивы объектов
>   https://skr.sh/sDXTxqSPVQo?a

Жаль что не скинули скриншоты сюда.
